src/options/build_mode.rs

```diff
@@ -19,6 +20,9 @@ impl Default for BuildMode {
 pub fn build_mode<'x>(ap: &mut ArgumentParser<'x>, mode: &'x mut BuildMode)
 {
     ap.refer(mode)
+    .add_option(&["--no-image"], StoreConst(BuildMode::NoImage), "
```

Comment:
Name is misleading. `--no-external-cache`, or `--skip-cache-url` or `--ignore-cache` or `--no-cache-check`, `--no-image-download`

We want to stress that it is a "cache" and it's external/network/download